charts/perses/values.yaml

```diff
@@ -154,12 +159,12 @@ config:
 sidecar:
   # -- Enable the sidecar container for ConfigMap provisioning
   enabled: false
-  image: 
+  image:
     # -- Container image repository for the sidecar
     repository: kiwigrid/k8s-sidecar
     # -- Container image tag for the sidecar
     tag: 1.30.0
-  # -- Label key to watch for ConfigMaps containing Perses resources  
+  # -- Label key to watch for ConfigMaps containing Perses resources
```

Comment:
Hey @dougkirkley can you rever this non related changes?

Reply:
He probably has an auto formatting removing trailing spaces, which is good in my opinion. Most of the IDE have it now no?

Reply:
To me this is fine, this is not changes that would conflict with another PR. A bit polluting that's true but going in the right direction

Reply:
@nicolastakashi  yeah, definitely an IDE setting. I can revert it if you really want.

Reply:
I'm fine too, 😄 
can you just bump the version on Chart.yaml @dougkirkley ?